WF-507:Dogfooding fixes (#7)
spatialos/FlexibleProjectExample: docs/migration-guide/faq.md

# FAQ

> **Is `spatial local worker launch` supported?**<br/>

Reviewer: ```suggestion
**Do you support `spatial local worker launch`?**<br/>
```
Passive -> active. Let me know if it's OK to say "you" here - hopefully it should be fairly clear what it means from the context.

Author: Nice! Done. :)

> `spatial local worker launch` is not supported and will not be supported going forward. Please start and connect your worker to a deployment manually as shown in the [readme](../../README.md#Running-the-project).

Reviewer: ```suggestion
We don't support `spatial local worker launch`and won't support it going forward. Please start and connect your worker to a deployment manually as shown in the [readme](../../README.md#Running-the-project).
```

Author: Done.

> **Is `spatial project history snapshot convert` supported?**<br/>

Reviewer: ```suggestion
**Do you support `spatial project history snapshot convert`?**<br/>
```
As above.

Author: Done.

> `spatial local worker launch` is not supported and will not be supported going forward. Please start and connect your worker to a deployment manually. For more information, please refer to our [API documentation](https://docs.improbable.io/reference/latest/csharpsdk/api-reference#improbable-worker-connection-class).

Reviewer 2: The reference to the docs is a bit misleading, since even with `spatial local worker launch`, the worker would have had to use the same thing. The only thing spatial local worker launch did was run the command with args specified in the worker configs.

`You will need to start your worker manually as shown here [insert link to the part of the FlexibleProjectExample README where we show the commands to start a worker]` ?

Author: Fixed.

> `spatial project history snapshot convert` is not supported at the moment. We are planning to provide this functionality in the future.

Reviewer: ```suggestion
We don't support `spatial project history snapshot convert` at the moment. We are planning to provide this functionality in the future.
```

Author: Done.

> `spatial project history snapshot convert` is not supported at the moment. We are planning to provide the functionality in the future.

Reviewer 2: ~the~ this

Author: Applied your suggestion. :)
